templates/modify-vm-template/manifests/modify-vm-template.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  annotations:
+    task.kubevirt.io/associatedServiceAccount: {{ sa_name }}
+    templateName.params.task.kubevirt.io/kind: {{ task_param_types.template_kind }}
+    templateName.params.task.kubevirt.io/apiVersion: {{ task_param_types.template_version }}
+    templateNamespace.params.task.kubevirt.io/kind: {{ task_param_types.namespace }}
+    cpuSockets.params.task.kubevirt.io/type: {{ task_param_types.string }}
+    cpuCores.params.task.kubevirt.io/type: {{ task_param_types.string }}
+    cpuThreads.params.task.kubevirt.io/type: {{ task_param_types.string }}
+    memory.params.task.kubevirt.io/type: {{ task_param_types.string }}
+  labels:
+    task.kubevirt.io/type: {{ task_name }}
+    task.kubevirt.io/category: {{ task_category }}
+  name: {{ task_name }}
+spec:
+  params:
```

Comment:
IIRC one of the requirements was changing the provider (owner/company) of the template. Are we still pursuing that?

Reply:
I am not aware of this rule. Do you remember where it was written?

Reply:
there is a vague description here: https://issues.redhat.com/browse/CNV-8469, but I guess you should sync with others to hear what is the current status on this

Reply:
+1, yes annotations should be enough then